### PR TITLE
Jakebayliss patch 2

### PR DESCRIPTION
**Pull request by jakebayliss**

### CODE REVIEW

File: rules/do-you-know-how-to-programmatically-get-git-commits/rule.md [hunk 1]
```diff
@@ -13,28 +13,24 @@ redirects: []
...
-With [Visual Studio Online](http://www.visualstudio.com/) now supporting Git, more developers are changing their source control repositories. What happens if an application you developed relies on the [TFS Client Object Model](http://msdn.microsoft.com/en-us/library/bb130146.aspx) to get information out of source control (e.g. changeset comments) and the developers start using Git?
+Using the [Azure DevOps API](https://docs.microsoft.com/en-us/rest/api/azure/devops/git/commits/get-commits?view=azure-devops-rest-6.0) you can programmatically get a list of commits from your repository with only a HTTP request.
 
-<!--endintro-->
 
-That's where the new [Visual Studio Online REST APIs](http://www.visualstudio.com/en-us/integrate/reference/reference-vso-overview-vsi.aspx) come in. You can get a list of commits from your VSO Git repository with only a HTTP request.
 
+![](devops-get-commits.png)
 
+ **Figure: HTTPS GET commits from your repository**
```
Comment (pierssinclairssw) [inline rules/do-you-know-how-to-programmatically-get-git-commits/rule.md:22]: Figure should be inside the [ ]

File: rules/do-you-know-how-to-programmatically-get-git-commits/rule.md [hunk 2]
```diff
@@ -13,28 +13,24 @@ redirects: []
...
 
-
-Using HTTPS with basic authentication, make a GET request to a URL as below, substituting in your VSO details. A JSON object will be returned. To quickly create classes from a JSON response, see the rule [Do you know how to easily get classes from a JSON response?](/do-you-know-how-to-easily-get-classes-from-a-json-response)
+Using HTTPS with basic authentication, make a GET request to a URL as below, substituting in your project details. A JSON object will be returned. To quickly create classes from a JSON response, see the rule [Do you know how to easily get classes from a JSON response?](/do-you-know-how-to-easily-get-classes-from-a-json-response)
 
 
 
 ![](8-08-2014-4-24-34-PM-compressor.png)
+
  **Figure: Using the Chrome extension Postman to execute our request with Basic Authentication**
```
Comment (pierssinclairssw) [inline rules/do-you-know-how-to-programmatically-get-git-commits/rule.md:32]: see above

**Review by pierssinclairssw — CHANGES_REQUESTED** [review]

File: rules/do-you-know-how-to-programmatically-get-git-commits/rule.md (hunk 1 above)
Comment (jakebayliss) [reply rules/do-you-know-how-to-programmatically-get-git-commits/rule.md]: Done

File: rules/do-you-know-how-to-programmatically-get-git-commits/rule.md (hunk 2 above)
Comment (jakebayliss) [reply rules/do-you-know-how-to-programmatically-get-git-commits/rule.md]: Done

**Review by pierssinclairssw — APPROVED** [review]

**Pull request merged**